sql-statements/sql-statement-explain.md

```diff
@@ -26,6 +26,24 @@ TiDB 支持 `EXPLAIN [options] FOR CONNECTION connection_id`，但与 MySQL 的
 
 ![ExplainableStmt](/media/sqlgram/ExplainableStmt.png)
```

Comment:
the images are still here 🤔

Reply:
Updated in https://github.com/pingcap/docs-cn/pull/5540/commits/4119ba2d72b724adf520f74aa6ef01c014eb8da2.